addons/mail/static/tests/discuss/core/web/messaging_menu.test.js

```diff
@@ -260,3 +267,33 @@ test("counter is updated on receiving message on non-fetched channels", async ()
     );
     await contains(".o-mail-MessagingMenu-counter", { text: "1" });
 });
+
+test("can use notification item swipe actions", async () => {
+    mockTouch(true);
+    patchUiSize({ size: SIZES.SM });
+    const pyEnv = await startServer();
+    const partnerId = pyEnv["res.partner"].create({ name: "Demo", email: "demo@odoo.com" });
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_type: "chat",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ partner_id: partnerId }),
+        ],
+    });
+    pyEnv["mail.message"].create({
+        author_id: partnerId,
+        body: "A message",
+        message_type: "comment",
+        model: "discuss.channel",
+        res_id: channelId,
+    });
+    await start();
+    await openDiscuss();
+    await contains("button.active", { text: "Inbox" });
```

Comment:
Note that in master, inbox is hidden for "handle by email" users and by default user in HOOT mail tests is "handle by email".

Should change assertion or set `notification_type: "inbox"` on self user to have inbox in master.